ngraph_bridge/ngraph_builder.cc

```diff
@@ -1086,141 +1086,24 @@ static Status TranslateConv2DOp(const Node* op,
   ng::CoordinateDiff ng_padding_below{0, 0};
   ng::CoordinateDiff ng_padding_above{0, 0};
 
-  Builder::MakePadding(tf_padding_type, ng_image_shape, ng_kernel_shape,
-                       ng_strides, ng_dilations, ng_padding_below,
-                       ng_padding_above);
+  ng::op::PadType ng_pad_type = ng::op::PadType::EXPLICIT;
+  if (tf_padding_type == "VALID") {
+    ng_pad_type = ng::op::PadType::VALID;
+  } else {
+    Builder::MakePadding(tf_padding_type, ng_image_shape, ng_kernel_shape,
+                         ng_strides, ng_dilations, ng_padding_below,
+                         ng_padding_above);
+  }
 
-  std::shared_ptr<ng::Node> ng_conv = ConstructNgNode<ng::op::Convolution>(
-      op->name(), ng_input, ng_filter, ng_strides, ng_dilations,
-      ng_padding_below, ng_padding_above);
+  std::shared_ptr<ng::Node> ng_conv = ConstructNgNode<ng::opset3::Convolution>(
+      op->name(), ng_input, ng_filter, ng_strides, ng_padding_below,
```

Comment:
Minor comment: To be consistent with the ngraph Opset3 params (& TF doc), consider renaming these vars like pads_begin, pads_end etc. Also, are the meanings still same e.g. pad_below == pad_begin?